src/helpers.js

```diff
@@ -1,4 +1,4 @@
-import capitalize from 'lodash/upperFirst';
+import * as R from 'ramda';
```

Comment:
Ditto. Please use named imports instead of `* as R`.

Reply:
Fixed.